.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -59,6 +59,6 @@ body:
     attributes:
       value: |
         > **Note:** Please note that you have to fill required fields to post an Issue.
-          If not please close or redo the issue and join our slack channel [wg-manifests](https://www.kubeflow.org/docs/about/community/) here. This link also contains our meeting schedule. 
+          If not please close or redo the issue and join our CNCF Slack Channel [kubeflow-platform](https://www.kubeflow.org/docs/about/community/) here. This link also contains our meeting schedule. 
```

Comment:
please use [#kubeflow-platfrom](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2) to link directly to the channel and also keep the link to https://www.kubeflow.org/docs/about/community/ as well. Please make it the same in the issue template and PR template in all places.